docs/conf.py

```diff
@@ -39,6 +39,10 @@
 #    'sphinx.ext.napoleon',  # alternative to numpydoc -- looks a bit worse.
 ]
 
+# See https://github.com/rtfd/readthedocs.org/issues/283
+mathjax_path = ('https://cdn.mathjax.org/mathjax/latest/MathJax.js?'
+                'config=TeX-AMS-MML_HTMLorMML')
```

Comment:
What about `mathjax_path = ('//:cdn.mathjax.org/...'` so it uses the same protocol RTD is viewed with? Hmm no, this would probably break viewing files locally over the `file://` protocol. Defaulting to https seems reasonable then.